doozerlib/cli/__main__.py

```diff
@@ -97,7 +97,7 @@ def print_version(ctx, param, value):
               expose_value=False, is_eager=True)
 @click.option("--data-path", metavar='PATH', default=None,
               help="Git repo or directory containing groups metadata")
-@click.option("--working-dir", metavar='PATH', default=None,
+@click.option("--working-dir", metavar='PATH', required=True,
```

Comment:
This does not seem to allow using the env var or config file to specify this?

Reply:
Should be good now.